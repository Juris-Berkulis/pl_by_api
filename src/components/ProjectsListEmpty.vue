<script setup lang="ts">
</script>

<template>
<div class="h-full w-full flex justify-center items-center">
    <p class="text-3xl font-bold underline">Список пуст</p>
</div>
</template>

<style scoped>
</style>
