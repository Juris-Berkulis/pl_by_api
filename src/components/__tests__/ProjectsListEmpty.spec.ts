import { mount } from "@vue/test-utils";
import { describe, expect, it } from "vitest";
import ProjectsListEmpty from "../ProjectsListEmpty.vue";

describe('ProjectsListEmpty', () => {
    const wrapper = mount(ProjectsListEmpty);

    it('Snapshot', () => {
        expect(wrapper).toMatchSnapshot()
    });

    it('Contains text', () => {
        expect(wrapper.text()).toContain('Список пуст');
    });
});
