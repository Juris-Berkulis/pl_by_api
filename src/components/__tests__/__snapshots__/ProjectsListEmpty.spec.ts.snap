// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ProjectsListEmpty > Snapshot 1`] = `
"<div class=\\"h-full w-full flex justify-center items-center\\">
  <p class=\\"text-3xl font-bold underline\\">Список пуст</p>
</div>"
`;
